### ISSUE TITLE
CM11 app doesn't show

### ISSUE THREAD

**Issue by kaaell** [domi007/silentSMS#5]
Hi,
I tried to install the apk but the app isn't showing...I used cm11 (signed and not). Could you suggest anything?
Regards, k.

**Comment by domi007** [comment]
@kaaell tell me the exact steps how you tried to install it please

**Comment by kaaell** [comment]
Hi, I use CM11 (4.4.2, galaxy s3) and I've followed your instructions, so:
#su
#mount -o rw,remount /system
#cp /storage/sdcard0/SilentSMS-signed.apk /system/app/SilentSMS-signed.apk (and, after, I tried with the not signed app)
#reboot
Best regards, k.

**Comment by domi007** [comment]
@kaaell
Hi, well this is weird...it should show up at least, because the signing keys didn't change. It is not compatible with 4.4 because the sendRawPDU function was removed so even if it would show up you would not be able to use its functionality.
However I'm planning on updating the app to support 4.4 with Xplosed modules, but I can't give you an ETA on that.

**Comment by kaaell** [comment]
Oh well...in the beginning I'd like to thanks you for your time. Now I do a downgrade to CM10.2 4.3.1 and the app comes to live but, when I try to send a sms, it crashes (com.android.phones). Is it linked to sendRawPDU function? Best regards, k.

**Comment by domi007** [comment]
Yes, exactly, it crashes because of that function (actually because of its absence :)).
